lib/request/middleware/index.js

```diff
@@ -3,13 +3,9 @@
 // Allows incoming requests to be pre-processed, transformed, or sent elsewhere
 'use strict';
 
-var apiTree, existslib, pathlib;
-
-pathlib = require('path');
-
-existslib = process.version.split('.')[1] === '6' && require('path') || require('fs');
-
-apiTree = require('apitree');
+var pathlib = require('path');
+var fs = require('fs');
+var apiTree = require('apitree');
```

Comment:
Is this automatic hinting? Looks like we need to add a `.jshintrc` file and define a node environment.